lib/remote/configobjectutility.cpp

```diff
@@ -352,12 +355,5 @@ bool ConfigObjectUtility::DeleteObjectHelper(const ConfigObject::Ptr& object, bo
 bool ConfigObjectUtility::DeleteObject(const ConfigObject::Ptr& object, bool cascade, const Array::Ptr& errors,
 	const Array::Ptr& diagnosticInformation, const Value& cookie)
 {
-	if (object->GetPackage() != "_api") {
-		if (errors)
-			errors->Add("Object cannot be deleted because it was not created using the API.");
-
-		return false;
-	}
```

Comment:
This check was moved to one call site of `DeleteObject()`, but it looks like the `remove-comment` and `remove-downtime` API actions now longer perform this check.

Reply:
In the `RemoveComment` and `RemoveDowntime` functions this check is already present [here](https://github.com/WuerthPhoenix/icinga2/blob/5aacae705fc17d8429b2e1e6d146ae9e218f3cee/lib/icinga/downtime.cpp#L312) and [here](https://github.com/WuerthPhoenix/icinga2/blob/5aacae705fc17d8429b2e1e6d146ae9e218f3cee/lib/icinga/comment.cpp#L192)